### PR TITLE
Fix a collection of bufr2ioda.x YAML files and test-ref files for naming conventions

### PR DESCRIPTION
**Pull request by gthompsnJCSDA**
## Description

This PR fixes about 10 ctests that broke with the merge into develop of bufr2ioda.x codes to keep working in the sprint branch with naming convention changes.  It also fixes the gsi_ncdiag.py code for 3 ctests to keep in sync with the naming convention changes.

### Issue(s) addressed

A separate one was not created.

## Acceptance Criteria (Definition of Done)

I anticipate that only 4 ctests in ioda-converters within sprint branch will remain broken for atmospheric composition (tropomi: 3 ctests and moppit (1 ctest).  With these changes all bufr2ioda.x and gsi_ncdiag.py tests should now be passing.

## Dependencies

- [ ] coordinate with [JCSDA-internal/ioda/pull/873](https://github.com/JCSDA-internal/ioda/pull/873)

### CODE REVIEW

**Review by srherbener — CHANGES_REQUESTED** [review]
Looks good. Just one simple change to get another test passing. Thanks!

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -1081,7 +1082,6 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_aeronet_aaod
 
 ecbuild_add_test( TARGET  test_${PROJECT_NAME}_airnow
                   TYPE    SCRIPT
-                  ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
```
[the remark this answers is not in the document]
Comment (gthompsnJCSDA) [reply test/CMakeLists.txt:1084]: > This change is causing one of the test failures `test_iodaconv_airnow`.


Ugh, a mistake on my part during a merge conflict.  I will fix it ASAP, thanks.

**Review by srherbener — APPROVED** [review]
Thanks for fixing the test.

**Review by srherbener — APPROVED** [review]
Thanks for fixing the test.

**Review by BenjaminRuston — APPROVED** [review]
yes let us move forward with this version, and find bugs from here forward

image

**Pull request merged**